build.gradle.kts

```diff
@@ -17,18 +17,13 @@ dependencyCheck {
 
 dependencies {
   implementation("com.beust:klaxon:5.5")
-  implementation("com.amazonaws:aws-java-sdk-s3:1.12.87")
-  implementation("io.sentry:sentry-spring-boot-starter:5.2.2")
-  implementation("net.javacrumbs.shedlock:shedlock-spring:4.28.0")
-  implementation("net.javacrumbs.shedlock:shedlock-provider-jdbc-template:4.28.0")
+  implementation("com.amazonaws:aws-java-sdk-s3:1.12.99")
+  implementation("io.sentry:sentry-spring-boot-starter:5.3.0")
+  implementation("net.javacrumbs.shedlock:shedlock-spring:4.29.0")
+  implementation("net.javacrumbs.shedlock:shedlock-provider-jdbc-template:4.29.0")
   implementation("nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect:3.0.0")
   implementation("org.apache.poi:poi-ooxml:5.0.0")
 
-  // These can be removed when (and only when) the dependencies have been updated in Spring 2.5.6. 9.0.53 has CVE's.
```

Comment:
This is now safe to remove as we are now picking up the updated transitive dependency via the update to the latest hmpps.gradle-spring-boot plugin.